src/features/PersonList/popularPeopleApi.js

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 
-const APIKey = "api_key=3af561a12389e6d632bf79207cb88b6c";
-const pathAPI = "https://api.themoviedb.org/3/";
+import { APIKey } from "../../APIKey";
+import { pathAPI } from "../../pathAPI";
```

Comment:
I've put them both in common/movieAPI on branch features/MovieDetails so we are gonna have a conflict. I don't mind your way. We just have to decide what we do and do the same :)

Reply:
ok. i dont change it now, we`ll decide after merging both branches